selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/swagger/SwaggerTest.java

```diff
@@ -43,6 +45,15 @@ public void setUp() throws Exception {
   @Test
   public void checkNameProjectTest() throws Exception {
     driver.navigate().to(swaggerUrl);
-    assertTrue(swagger.getWsNamesFromWorkspacePage().contains(workspace.getName()));
+
+    try {
+      assertTrue(swagger.getWsNamesFromWorkspacePage().contains(workspace.getName()));
+    } catch (IllegalStateException ex) {
+      // remove try-catch block after issue has been resolved
+      Assert.fail("Known issue https://github.com/eclipse/che/issues/6015");
+    } catch (StaleElementReferenceException ex) {
```

Comment:
You can combine IllegalStateException and StaleElementReferenceException into the one catch() section:
} catch (IllegalStateException | StaleElementReferenceException ex) {
      // remove try-catch block after issue has been resolved
      Assert.fail("Known issue https://github.com/eclipse/che/issues/6015");
}